wp/lib/bap_wp/src/postprocess.ml

```diff
@@ -0,0 +1,133 @@
+(***************************************************************************)
```

Comment:
Not crazy about the name for this module. Maybe "output.ml"?

Reply:
Or even "output_gdb.ml", I'm not actually sure how good an idea it is to reuse this module for other things.

Reply:
I did also move the print_result function over into here. If I'm going to name it output_gdb.ml I should move that back. output.ml is perfectly acceptable